### ISSUE TITLE
[RFC]: Add bundle documentations to website docs

### ISSUE THREAD

**Issue by Planeshifter** [stdlib-js/www#60]
### Description

This RFC proposes to add the tailored documentation for the various bundles ([ESM](https://github.com/stdlib-js/string-constantcase/blob/esm/README.md), [Deno](https://github.com/stdlib-js/string-constantcase/blob/deno/README.md), [UMD](https://github.com/stdlib-js/string-constantcase/blob/esm/README.md)) that we have inside each standalone package repository to be available on the website documentation as well.

### Related Issues

No.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

**Comment by kgryte** [comment]
Agreed. This would be quite nice to have, but also we'd need to think about how we'd need to update the build process to make this happen. Likely, we'd need to re-design from scratch.

**Comment by Rupa-Rd** [comment]
Is this issue resolved already? Or should I work on it?

**Comment by kgryte** [comment]
It is still open; however, you should first study the build process which currently lives in the main project repo, not in this repo.

**Comment by Rupa-Rd** [comment]
Just to clarify things, the README file for the bundles (ESM and Deno) is present inside each repo. So, the aim of this issue is to display those documents in the main repo (stdlib)? For example, for string concatase the esm docs are present [here](https://github.com/stdlib-js/string-constantcase/blob/esm/README.md) but only the main README file of concatase is displayed on the main website. Correct me if I am wrong.

**Comment by kgryte** [comment]
As I mentioned, the build process happens in the main project repository. In that repo, we use those READMEs. We do NOT use the READMEs in the standalone repositories.
